### DISCUSSION TITLE
Is the District list is Tenant specific or Instance specific?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#446] [General Discussion]
District drop-down only shows Bengaluru Urban
 Can we have all 34 districts of Karnataka?
Because on "**DIKSHA**" we have all districts from particular states. (Refer to screenshots)
**The below image is from Sunbird Saas**
![image](https://user-images.githubusercontent.com/109504832/228223821-52873ed9-6122-4522-8f55-7a6c5c8d17eb.png)
 
**The below image is from Diksha. Can we create like this for Sunbird Saas. If "yes" Then How?** 
![image](https://user-images.githubusercontent.com/109504832/228222927-99fb0129-0083-4f67-a59a-8d3ea8233313.png)

[@vinukumar-vs](https://github.com/vinukumar-vs)  [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by swayangjit** [comment]
@sachinb80 [This thread](https://github.com/orgs/Sunbird-Ed/discussions/448)  will resolve your query.

**Reply by sachinb80** [reply]
Sure @swayangjit and thanks for the reply
